### PR TITLE
MGMT-4289 Add e2e-metal-assisted-none-platform jobs

### PR DESCRIPTION
**Pull request by YuviGold**

### CODE REVIEW

File: ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml [hunk 1]
```diff
@@ -65,6 +65,52 @@ tests:
...
         PROXY=yes
     workflow: baremetalds-assisted-install
+- as: e2e-metal-assisted-none-platform
+  steps:
+    cluster_profile: packet
+    dependencies:
+      ASSISTED_SERVICE_IMAGE: pipeline:assisted-service
+    env:
+      ASSISTED_CONFIG: |
+        PLATFORM=none
```
Comment (ori-amizur) [inline ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:75]: Aren't you trying to install with DHCP?  I don't think it is supported for none platform

Comment (YuviGold) [reply ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:75]: DHCP is enabled by default

**Comment by YuviGold** [comment]
none platform IPv4 passes. IPv6 fails but it would be fixed on a different PR. @ori-amizur 
/unhold

**Comment by ori-amizur** [comment]
LGTM

**Pull request merged**